Server/ConcordiaCurriculumManager/Repositories/DossierRepository.cs

```diff
@@ -92,6 +92,8 @@ public async Task<List<Dossier>> GetDossiersByID(Guid userId)
         .ThenInclude(a => a.Group == null ? null : a.Group.Members)
         .Include(d => d.ApprovalStages)
         .ThenInclude(a => a.Group == null ? null : a.Group.GroupMasters)
+        .Include(d => d.ApprovalHistories)
+        .ThenInclude(a => a.Group)

```

Comment:
Before merging, can you double check this query works correctly? Because in the CCMDbContext.cs file you never specified the relationship between approval histories and groups, so I'm concerned it might crash on loading something like the dossier details page.

Reply:
Yep, I checked and there's no issue on the page.